apps/bfd-pipeline/bfd-pipeline-shared-utils/src/main/java/gov/cms/bfd/pipeline/sharedutils/DatabaseOptions.java

```diff
@@ -10,7 +9,6 @@
   private final String databaseUrl;
   private final String databaseUsername;
   private final char[] databasePassword;
-  private final DataSource databaseDataSource;
```

Comment:
Moving dataSource out of this class is a huge win.  Thanks for doing that.  Now the class is (mostly) immutable as it should be as a config.  I'd much rather see the password as a String rather than array so the whole object can be truly immutable.

Reply:
Thanks! Done.